test/current_toolchain_tools/current_exec_files_test.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -xeuo pipefail
```

Comment:
Did you leave -x there intentionally? (I typically only use it for debugging)

Reply:
I thought it was handy for debugging, yeah, but it can be removed.